### ISSUE TITLE
Edge: web sites with application role do not work as intended

### ISSUE THREAD

**Issue by PratikP1** [nvaccess/nvda#6949]
Web applications such as Google Inbox and Wunderlist with well defined application roles do not work properly with NVDA in Microsoft Edge. The application fails to obtain and retain focus. When focus is obtained, defined keyboard commands do not work.

STR: 
1. Start Microsoft Edge
2. Open the address bar by pressing alt+d.
3. Visit inbox.google.com.
4. If you are not logged in to your gmail account, you will be asked to do so.
5. Go to the main area by going to the main landmark. Press d.
6. use the down arrow key to arrive in the application area.
7. Press enter to enter focus mode.
8. Press NVDA+spacebar to force focus mode.
9. Try pressing left and right arrow keys to read list of messages or bundles.

Test environment: Windows 10 Pro build 15048
NVDA version next-13899,af44c1d4

**Comment by jcsteh** [comment]
CC @MichaelDCurran

**Comment by michaelDCurran** [comment]
Steps 7 and 8 are contradictory. Also, What results do you actually get?  If I arrow down to the main landmark And lock focus mode with NVDA+space (note not pressing enter first) then I can use the arrow keys to move between the conversations etc and NVDA announces the focus changes. If this is not the result you got could you please try with a latest NVDA next build?

**Comment by PratikP1** [comment]
I should have written steps 7 and 8 more clearly. I should have said that pressing enter does not allow user to enter into the application. If it doesn’t work, try nvda+spacebar.

With Insider build 15063 and latest NVDA next, this aspect works. I can now enter the application mode. I am having trouble disengaging focus mode. There are other issues. I’m not certain whether they’re related to how Inbox is designed. For instance, the built-in keyboard commands should activate an announcement of their action such as delete, mark message as done, etc.

**Comment by Adriani90** [comment]
This works as expected in Edge Chromium 81. Closing as works for me.
